src/datepicker/datepicker.js

```diff
@@ -19,7 +19,8 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
   shortcutPropagation: false,
   showWeeks: true,
   yearColumns: 5,
-  yearRows: 4
+  yearRows: 4,
+  monthColumns: 4
```

Comment:
Default number it seems to replace is 3 - this is probably what is breaking the unit tests.